README.md

```diff
@@ -2,7 +2,11 @@
 
 [![npm version](https://badge.fury.io/js/rendition.svg)](http://badge.fury.io/js/rendition)
 [![Build Status](https://travis-ci.org/resin-io-modules/rendition.svg?branch=master)](https://travis-ci.org/resin-io-modules/rendition)
+[![Dependency Status](https://img.shields.io/david/resin-io-modules/rendition.svg)](https://david-dm.org/resin-io-modules/rendition)
+[![npm](https://img.shields.io/npm/dm/rendition.svg?maxAge=2592000)](https://www.npmjs.com/package/rendition)
+[![GitHub last commit](https://img.shields.io/github/last-commit/resin-io-modules/rendition.svg)](https://github.com/resin-io-modules/rendition/commits/master)
 [![style: styled-components](https://img.shields.io/badge/style-%F0%9F%92%85%20styled--components-orange.svg?colorB=daa357&colorA=db748e)](https://github.com/styled-components/styled-components)
+[![Join the chat at https://gitter.im/resin-io-modules/rendition](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/resin-io-modules/rendition?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
This is quite a few badges, and when they end up on https://www.npmjs.com/package/rendition you'll have less space than you do on github, so I think you're breaking this out across multiple lines of badges there. Not a huge problem, but it might be good to trim these down a little.

Reply:
Sure, I'd like to go ahead with the badges in this PR, and then we can trim them back at a later point

Reply:
On second thoughts, I'm going to kill off the 'downloads' and 'last commit' badges